### ISSUE TITLE
Our systems have detected unusual activity from your device. Please refresh your browser and try again later.

### ISSUE THREAD

**Issue by Valmach** [josStorer/chatGPTBox#681]
**Describe the bug**
**问题描述**
Unusual activity..   

**To Reproduce**
**如何复现** duckduckgo
2. Click on '....'chatgptbox
3. Scroll down to '....' error

**Expected behaviour**
**期望行为**

Output from manual search query

**Screenshots**
![how-can-i-file-a-law-suit-againt-the-us-government-for-genocide-at-DuckDuckGo](https://github.com/josStorer/chatGPTBox/assets/1754881/0b5d5c89-7cf3-4f31-b4d2-19111ee25a35)

**截图说明**

**Please complete the following information):**
**请补全以下内容**
 - OS: [e.g. Windows]  Windows and  Linux  on Chrome
 - Browser: [e.g. chrome, safari] Chrome Brave, Firefox, Edge,  
 - Extension Version: [e.g. v2.0.2]

**Additional context**
**其他**

**Comment by Aerasyn** [comment]
Yep, been doing this for a month or more now. I thought it was related to signing out then back into OpenAI site, but that has stopped being a consistant fix anymore. A shame. I really like this extension. Seems like an issue on OpenAI's side, like they're trying to fight 3rd party software from using it in this way or something. Hope it gets resolved.

**Comment by Jerusalem111** [comment]
> I really like this extension. 

Same here.  This guy has the best extention on all of firefox.  His extention transformed my life.  He is a good man.  May God bless him for his time and effort.

**Comment by duAffentier** [comment]
Here are the same!
With NordVPN and without VPN!
It is dosent Work since the weekend!

**Comment by gitdine** [comment]
> Yep, been doing this for a month or more now. I thought it was related to signing out then back into OpenAI site, but that has stopped being a consistant fix anymore. A shame. I really like this extension. Seems like an issue on OpenAI's side, like they're trying to fight 3rd party software from using it in this way or something. Hope it gets resolved. I've switched the API mode to Gemini since few of the others work anymore. Works well enough for the summarize things I do with the extension.

Yep same here, switched to Gemini for now. Works well enough, only problem is 'disable history' feature doesn't work for Gemini/Claude so I have to go and manually delete all the spammy entries that get added to my chat history.

**Comment by chrahman** [comment]
Same issue from chatgpt, anyone have solution?

**Comment by Jerusalem111** [comment]
I just paid for a gpt4 subscription but its limited to like 30 or so prompts every 3 hrs. And its slow.

**Comment by josStorer** [comment]
fixed in https://github.com/josStorer/chatGPTBox/commit/77a14e23ef47dbbcf3419350db10ceefa8322eaf

**Comment by Hatsu-no** [comment]
> fixed in [77a14e2](https://github.com/josStorer/chatGPTBox/commit/77a14e23ef47dbbcf3419350db10ceefa8322eaf)

Hello, I have updated the ChatGPT BOX, it's not working for.
I am deeply grateful for your work, and this had helped me a lot on the past year. I hope it comes back online quickly!!!

**Comment by josStorer** [comment]
@Hatsu-no I will release a version update later, and then you will need to wait for the store review before you can get this update.

**Comment by Hatsu-no** [comment]
> @Hatsu-no I will release a version update later, and then you will need to wait for the store review before you can get this update.

thank you so much ♥♥

**Comment by josStorer** [comment]
[v2.5.3](https://github.com/josStorer/chatGPTBox/releases/tag/v2.5.3)

**Comment by gitdine** [comment]
Updated to 2.5.3 and it's working perfectly.
Thanks so much @josStorer for your hard work on this amazing project.

**Comment by duAffentier** [comment]
Hi,

![grafik](https://github.com/josStorer/chatGPTBox/assets/149059068/bdce3b88-a59a-4584-8216-b4c3bbda1df5)

I have the new version, because this problem - no answer...

**Comment by Aerasyn** [comment]
Hmm, 2.5.3 was working for a couple days but now it looks like openai changed the website url to chatgpt.com. I had to change my speed dial bookmarks and my pinned tab. Derp. The extenstion has started giving the same unusual activity warning again. Hope it gets resolved. Back to Gemini as the option for now again ;)

**Comment by duAffentier** [comment]
Now:

Fehler:

Our systems have detected unusual activity from your device. Please refresh your browser and try again later.
![grafik](https://github.com/josStorer/chatGPTBox/assets/149059068/2804eb07-f823-42a6-9f96-aa3426dd930b)

**Comment by duAffentier** [comment]
@josStorer - It worked briefly, but now the same error messages again!
Geminin is doing very well! But ChatGPT no longer works!

**Comment by Valmach** [comment]
As stated in previous reporting. The Extension stopped to function after upgrading OpenAI... A potential move forward…   After resetting IPV4 setting … Clearing all OpenAI cookies, then removing and reinstalling ChatGptBox extension.   The error....' unusual activity has been detected' has stopped.... now just blinking, but now responding to 'search query'  my options set to manual. 
![ipv42024-05-06 030552](https://github.com/josStorer/chatGPTBox/assets/1754881/fbc26389-9dfa-4c9a-9044-04a2e4f031be)
![Error-Our-systems-have-detected-unusual-activity-from-your-device-Please-refresh-your-browser-and-try-again-later-at-DuckDuckGo](https://github.com/josStorer/chatGPTBox/assets/1754881/cec33667-a11f-4c8b-9671-a28fb3362428)

**Comment by josStorer** [comment]
https://github.com/josStorer/chatGPTBox/releases/tag/v2.5.4

**Comment by gitdine** [comment]
Can confirm it's working now. Thanks for the fix!

**Comment by duAffentier** [comment]
Hi
with NordVPN more Problems!
New Firefox installation, one day its works, the next day same failure...

**Comment by josStorer** [comment]
@duAffentier  Open chatgpt.com and try again

**Comment by duAffentier** [comment]
Hi! actually other Problems with the API Key... :-(

**Comment by PeterDaveHello** [comment]
Replied and closed by gpt-5.6-luna xhigh:

> This duplicates the earlier #677, which tracks the same unusual-activity error and its v2.5.4 fix. Keeping #677 as the original report.
